src/ArmTemplates/Commands/Configurations/ExtractorConsoleAppConfiguration.cs

```diff
@@ -92,5 +92,11 @@ public class ExtractorConsoleAppConfiguration
 
         [Option(longName: "extractGateways", HelpText = "Attempt to extract information about Self-Hosted gateways")]
         public string ExtractGateways { get; set; }
+
+        [Option(longName: "overrideGroupNames", HelpText = "Override group GUID identification to system generated")]
```

Comment:
Also I would change name to `OverrideGroupGuids` and `OverrideProductGuids`, because it can be confusing, but it is not that important if there is documentation on that